help/destinations/catalog/streaming/pegaCDH.md

```diff
@@ -0,0 +1,116 @@
+
+## Prerequisites {#prerequisites}
+
+Before you use Pega Customer Decision Hub destination to export data out of Adobe Experience Platform:
+
+* Configure the Adobe Segment Membership Component in your Pega Customer Decision Hub instance.
+* Configure OAuth 2.0 Client Registration using Client Credentials grant type in your Pega Customer Decision Hub instance
+* Configure real-time processing of Adobe Segment Membership data flow to run in your Pega Customer Decision Hub instance.
+
+## Supported identities {#supported-identities}
+
+Pega Customer Decision Hub supports the activation of custom user ID described in the table below. For more details, see [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Description
+|---|---|
+|*CustomerID*|Common User Identifier that uniquely identifies a profile in Pega Customer Decision Hub and Adobe Experience Platform
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | Export all members of a segment with identifier (*CustomerID*), attributes (last name, first name, location,..) and Segment Membership data.|
+| Export frequency | **[!UICONTROL Streaming]** | Streaming destinations are an always on API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. For more information, see [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+Follow the steps described  to connect to this destination. In the configure destination workflow, fill in the fields listed in the two sections below.
+
+### Authenticate to destination {#authenticate}
+
+#### OAuth 2 Client Credentials authentication {#oauth-2-client-credentials-authentication}
+
+Fill in the fields below and select **[!UICONTROL Connect to destination]**:
+
+* **[!UICONTROL Access Token URL]**: The URL on your side which issues access token.
+* **[!UICONTROL Client ID]**: The [!DNL client ID] that your system assigns to Adobe Experience Platform.
+* **[!UICONTROL Client Secret]**: The [!DNL client secret] that your system assigns to Adobe Experience Platform.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required fields and select **[!UICONTROL Next]**.
```

Comment:
Same comment as above. Can you please add a screenshot with the three fields filled in?

Reply:
Added